pkg/apis/pravega/v1alpha1/types.go

```diff
@@ -33,9 +33,10 @@ type PravegaCluster struct {
 }
 
 type PravegaClusterSpec struct {
-	ZookeeperUri string         `json:"zookeeperUri"`
-	Bookkeeper   BookkeeperSpec `json:"bookkeeper"`
-	Pravega      PravegaSpec    `json:"pravega"`
+	ZookeeperUri   string         `json:"zookeeperUri"`
+	ExternalAccess bool           `json:"externalAccess"`
```

Comment:
I would strongly suggest that `externalAccess` be modeled as a struct, maybe with `enabled` field.   Over time there will be more parameters related to this.

Reply:
I like the idea of making external access configurable. I'll create a struct as you suggested.

Reply:
Updated. I've created a struct for the external access configuration.